services/director-v2/tests/unit/with_dbs/test_api_route_computations.py

```diff
@@ -313,6 +378,229 @@ async def test_create_computation(
     assert response.status_code == status.HTTP_201_CREATED, response.text
 
 
+@pytest.fixture
+def wallet_info(faker: Faker) -> WalletInfo:
+    return WalletInfo(wallet_id=faker.pyint(), wallet_name=faker.name())
+
+
+@pytest.fixture
+def fake_ec2_cpus() -> PositiveInt:
+    return 4
+
+
+@pytest.fixture
+def fake_ec2_ram() -> ByteSize:
+    return parse_obj_as(ByteSize, "4GiB")
+
+
+@pytest.fixture
+def mocked_clusters_keeper_service_get_instance_type_details(
+    mocker: MockerFixture,
+    default_pricing_plan_aws_ec2_type: str,
+    fake_ec2_cpus: PositiveInt,
+    fake_ec2_ram: ByteSize,
+) -> mock.Mock:
+    return mocker.patch(
+        "simcore_service_director_v2.modules.db.repositories.comp_tasks._utils.get_instance_type_details",
+        return_value=[
+            EC2InstanceType(
+                name=default_pricing_plan_aws_ec2_type,
+                cpus=fake_ec2_cpus,
+                ram=fake_ec2_ram,
+            )
+        ],
+    )
+
+
+@pytest.fixture
+def mocked_clusters_keeper_service_get_instance_type_details_with_invalid_name(
+    mocker: MockerFixture,
+    faker: Faker,
+    fake_ec2_cpus: PositiveInt,
+    fake_ec2_ram: ByteSize,
+) -> mock.Mock:
+    return mocker.patch(
+        "simcore_service_director_v2.modules.db.repositories.comp_tasks._utils.get_instance_type_details",
+        return_value=[
+            EC2InstanceType(
+                name=faker.pystr(),
+                cpus=fake_ec2_cpus,
+                ram=fake_ec2_ram,
+            )
+        ],
+    )
+
+
+@pytest.fixture(params=ServiceResourcesDictHelpers.Config.schema_extra["examples"])
+def project_nodes_overrides(request: pytest.FixtureRequest) -> dict[str, Any]:
+    return request.param
+
+
+async def test_create_computation_with_wallet(
+    minimal_configuration: None,
+    mocked_director_service_fcts: respx.MockRouter,
+    mocked_catalog_service_fcts: respx.MockRouter,
+    mocked_resource_usage_tracker_service_fcts: respx.MockRouter,
+    mocked_clusters_keeper_service_get_instance_type_details: mock.Mock,
+    product_name: str,
+    fake_workbench_without_outputs: dict[str, Any],
+    registered_user: Callable[..., dict[str, Any]],
+    project: Callable[..., Awaitable[ProjectAtDB]],
```

Comment:
Rename `create_project` or `insert_project`? (See `CC3: Naming fixtures that return Callables`)

Same in the others tests below

Reply:
ahlala... yes. if that means refactoring all the test then no. will check

Reply:
so then @pcrespov it's no. this fixture comes from the pytest-simcore package. I can create a separate PR for this. but otherwise it will just shadow anything else.

Reply:
no. for the same reason. if I get some free time, I might create a separate PR for this. but this would just pollute this PR.